app/controllers/api/functions.php

```diff
@@ -202,6 +204,12 @@
             throw new Exception(Exception::GENERAL_ARGUMENT_INVALID, 'When connecting to VCS (Version Control System), you need to provide "installationId" and "providerBranch".');
         }
 
+        $spec = Config::getParam('runtime-specifications')[$specification] ?? [];
+
+        if (empty($spec) || empty($spec['memory']) || empty($spec['cpus'])) {
+            throw new Exception(Exception::FUNCTION_INVALID_RUNTIME_SPECIFICATION);
+        }
+
```

Comment:
This should not be needed, ideally. Ideally validator checks all of this

Reply:
Remove